optimade/server/routers/versions.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import Request
+from fastapi.responses import PlainTextResponse
+from starlette.routing import Router, Route
+
+from .utils import BASE_URL_PREFIXES
+
+
+async def versions(request: Request):
+    """ Respond with the text/csv representation for the served versions. """
+    version = BASE_URL_PREFIXES["major"].replace("/v", "")
+    response = f"version\n{version}"
+    return PlainTextResponse(
+        response, media_type="text/csv", headers={"header": "present"}
+    )
+
+
+router = Router(routes=[Route("/versions", endpoint=versions)])
```

Comment:
Move to top of file (under imports) to replicate structure of the other router files.

Reply:
This now uses the API router